lib/LedgerSMB/Scripts/setup.pm

```diff
@@ -389,45 +389,47 @@ sub run_backup {
     my $backupfile;
     my $mimetype;
 
-    if ($request->{backup} eq 'roles'){
-       my @t = localtime(time);
-       $t[4]++;
-       $t[5] += 1900;
-       $t[3] = substr( "0$t[3]", -2 );
-       $t[4] = substr( "0$t[4]", -2 );
-       my $date = "$t[5]-$t[4]-$t[3]";
-
-       $backupfile = $database->backup_globals(
-                      tempdir => $LedgerSMB::Sysconfig::backupdir,
-                         file => "roles_${date}.sql"
-       );
-       $mimetype   = 'text/x-sql';
-    } elsif ($request->{backup} eq 'db'){
-       $backupfile = $database->backup;
-       $mimetype   = 'application/octet-stream';
-    } else {
+    if ($request->{backup} eq 'roles') {
+        my @t = localtime(time);
+        $t[4]++;
+        $t[5] += 1900;
+        $t[3] = substr( "0$t[3]", -2 );
+        $t[4] = substr( "0$t[4]", -2 );
+        my $date = "$t[5]-$t[4]-$t[3]";
+
+        $backupfile = $database->backup_globals(
+            tempdir => $LedgerSMB::Sysconfig::backupdir,
+            file => "roles_${date}.sql"
+        );
+        $mimetype = 'text/x-sql';
+    }
+    elsif ($request->{backup} eq 'db') {
+        $backupfile = $database->backup;
+        $mimetype   = 'application/octet-stream';
+    }
+    else {
         die $request->{_locale}->text('Invalid backup request');
     }
 
     $backupfile or
         die $request->{_locale}->text('Error creating backup file');
 
-    if ($request->{backup_type} eq 'email'){
+    if ($request->{backup_type} eq 'email') {
         # suppress warning of single usage of $LedgerSMB::Sysconfig::...
         no warnings 'once';
```

Comment:
Probably don't want a blank line after this to help group it with the statement it affects.
Ideally wrap the 2 statements in something like....
```plain
# ##########
# suppress warning .......
no warnings 'once';
assignment to suppress warnings for;
# ##########
```

@ehuelsmann However, I wonder why we need to suppress the warnings here.
If we really do need to suppress them, then I'd like to see a comment describing the reason.